arangosh/Restore/RestoreFeature.cpp

```diff
@@ -755,6 +815,15 @@ int RestoreFeature::processInputDirectory(std::string& errorMsg) {
                   << " collection(s), read " << _stats._totalRead << " byte(s) from datafiles, "
                   << "sent " << _stats._totalBatches << " data batch(es) of " << _stats._totalSent << " byte(s) total size" << std::endl;
       }
+
+      if (didModifyFoxxCollection) {
+        // if we get here we need to trigger foxx heal
+        Result res = ::triggerFoxxHeal(*_httpClient);
+        if (res.fail()) {
+          LOG_TOPIC(WARN, Logger::RESTORE) << "Reloading of Foxx failed. In the cluster Foxx Services will be available eventually, On SingleServers send a PUT to ";
```

Comment:
0o this is my fail. We forgot to add the route: 
`Send a POST request to /_api/foxx/_local/heal on the current database with an empty body.`